pages/wizard/rhev/configuration.py

```diff
@@ -0,0 +1,64 @@
+from selenium.webdriver.common.by import By
+from pages.base import Base
+
+
+class Configuration(Base):
+    _page_title = "QuickStart Cloud Installer"
+    _root_password_field_locator = (By.ID, "rhev-root-password")
+    _confirm_root_password_field_locator = (By.ID, "confirm-rhev-root-password")
+    _engine_admin_password_field_locator = (By.ID, "rhev-engine-admin-password")
+    _confirm_engine_admin_password_field_locator = (By.ID, "confirm-rhev-engine-pdmin-password")
+    _data_center_name_field_locator = (By.ID, "rhev-data-center-name")
+    _cluster_name_field_locator = (By.ID, "rhev-cluster-name")
+    #TODO
+    _cpu_type_dropdown_locator = ()
+
+    @property
+    def root_password_field(self):
+        return self.selenium.find_element(*self._root_password_field_locator)
+
+    @property
+    def confirm_root_password_field(self):
+        return self.selenium.find_element(*self._confirm_root_password_field_locator)
+
+    @property
+    def engine_admin_password_field(self):
+        return self.selenium.find_element(*self._engine_admin_password_field_locator)
+
+    @property
+    def confirm_engine_admin_password_field(self):
+        return self.selenium.find_element(*self._confirm_engine_admin_password_field_locator)
+
+    @property
+    def data_center_name_field(self):
+        return self.selenium.find_element(*self._data_center_name_field_locator)
+
+    @property
+    def cluster_name_field(self):
+        return self.selenium.find_element(*self._cluster_name_field_locator)
+
+    #TODO
+    @property
+    def cpu_type_dropdown(self):
+        return self.selenium.find_element(*self._cpu_type_dropdown_locator)
+    
+    
+    def set_root_password(self, password):
```

Comment:
We should not clear the password I don't think...in the configuraiton page for CloudForms I have a clear method and a set method...but maybe thats not a bad idea...its just there could be a test that wanted to add to something already there.

Definitely we should have separate set_root_password() and set_confirm_root_passwd(), just so we can test setting them different.